shared/src/bundles/user-details/user-details.ts

```diff
@@ -0,0 +1,17 @@
+export {
+    CountryList,
+    EmploymentType,
+    EnglishLevel,
+    JobTitle,
+    NotConsidered,
+    PreferredLanguages,
+    UserDetailsApiPath,
+} from './enums/enums.js';
+export {
+    type UserDetailsApproveRequestDto,
+    type UserDetailsResponseDto,
+    type UserDetailsUpdateDto,
+    type UserDetailsUpdateRequestDto,
+} from './types/types.js';
+export { userDetailsUpdate as userDetailsUpdateValidationSchema } from './validation-schemas/validation-schemas.js';
+export { userDetailsApprove as userDetailsApproveValidationSchema } from './validation-schemas/validation-schemas.js';
```

Comment:
You don't need this in this PR. This is another task. Please don't go beyond the scope of your task !